.github/workflows/build.yml

```diff
@@ -98,3 +98,63 @@ jobs:
       - name: Setup tmate debug session
         uses: mxschmitt/action-tmate@v3
         if: env.RUN_TMATE
+  test:
+    name: test source - go${{ matrix.go-version }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
+        go-version:
+          - "1.19"
+          - "1.20"
+        include:
+          - os: ubuntu-20.04
+            go-version: "1.20"
```

Comment:
Is this necessary? The only reason we have such a thing in [cisagov/skeleton-python-library](https://github.com/cisagov/skeleton-python-library/blob/ea4912cef0f380adaf030ca8d8df5cfc3696bcbf/.github/workflows/build.yml#L119-L121) is because they will not make a Python 3.6 package available for Ubuntu 22.04 (`ubuntu-latest`). There should be no need for it here as long as the Go versions being tested have packages available for the `ubuntu-latest` runner tag.

```suggestion
```